index.html

```diff
@@ -18,32 +18,27 @@
       <img id="oldagram-logo" src="./images/logo.png" alt="oldagram logo" />
       <img id="user-avatar" src="./images/user-avatar.png" alt="user avatar" />
     </header>
-    <section id="poster-info-section">
-      <img
-        id="poster-avatar"
-        src="./images/avatar-vangogh.jpg"
-        alt="vangogh avatar"
-      />
-      <p id="poster-name">
-        Vincent van Gogh <span id="poster-location">Zudert, Netherlands</span>
-      </p>
-    </section>
-    <section id="img-section">
-      <img
-        id="poster-image"
-        src="./images/post-vangogh.jpg"
-        alt="self portrait painting of Vangogh"
-      />
-    </section>
-    <section id="poster-message-section">
-      <img class="icon" src="./images/icon-heart.png" alt="heart icon" />
-      <img class="icon" src="./images/icon-comment.png" alt="comment icon" />
-      <img class="icon" src="./images/icon-dm.png" alt="message icon" />
-      <p class="poster-message-section-text" id="post-likes">21,492 likes</p>
-      <p class="poster-message-section-text" id="poster-message">
-        <span id="poster-username">vincey1863</span> just took a few mushrooms
-        lol
-      </p>
-    </section>
+    <div id="posts">
```

Comment:
moved everything below the header into a div to leverage the .innerHTML method when generating the html from the posts array